### ISSUE TITLE
Investigate why rejection is not always happening when RAM goes over limit 

### ISSUE THREAD

**Issue by sabrinasadik** [threefoldtech/zos#1275]
Another node went down last night at 3.20AM. RAM was at 14.6GB out of 16GB available. K8S reservation for 2GB came it, which the node should have rejected but it didn't. Need to investigate why.

Node Hx4B6B8s8yQ5ABLStma1kZ33kCuprSxzU8tY4totKbWH
See original issue https://github.com/threefoldtech/itenv_testnet/issues/9

**Comment by muhamadazmy** [comment]
![image](https://user-images.githubusercontent.com/10920323/119100854-ab3fd200-ba18-11eb-9ea9-ce9e2298ed73.png)

That is screenshot from the node metrics:
- The gap is information is the time the node was down.
- We notice the following:
  - Before node crash there is increase in used memory -yellow- (around 2GB)  this increase is not associated with any **new** workload, so this increase is not related to capacity planning. The increase happened because of an already running process.
  - The increase, is **not** the cause of the crash, because you can see the node still have a lot of memory allocated to cache+buffer (gray) this part of the memory is used by the kernel to speed up file access, but still will get freed up if needed by the system or other processes. In other words, this part of memory is considered (available)
  - We notice also that the node operates normally for (30 min) from after the 2GB increase in used memory until the crash

So to sum up:
- There was (for over 12 hours) no new reservations deployed on this node. Hence no direct issue with memory capacity planning.
- There was plenty of usable memory on the node the moment of crash
- I am not saying it's not a "memory" problem, but I am saying it's not a "memory planning" problem, we did not allow new reservations on the node, and the node did not deploy any new workloads that is memory hungry before the crash by 12 hours.

All this makes me really think that this is not a memory capacity related issue. Something else went rogue on the node and caused it to crash.

I am still looking into what might be the problem
